packages/rocketchat-ui-sidenav/client/sidebarItem.js

```diff
@@ -95,7 +95,17 @@ Template.sidebarItem.onCreated(function() {
 });
 
 Template.sidebarItem.events({
-	'click [data-id], click .sidebar-item__link'() {
+	'click [data-id], click .sidebar-item__link'(e) {
+		const element = e.currentTarget;
+		const sidebarElements = document.getElementsByClassName('sidebar-item');
+		for (let i = sidebarElements.length - 5; i < sidebarElements.length; i++) {
```

Comment:
We shouldn't rely on magic constants in expressions like `sidebarElements.length - 5`. What about using a more scoped query selector get the items?

```
const sidebarElements = document.querySelectorAll('.sidebar .flex-nav .sidebar-item');
for (const sidebarElement of Array.from(sidebarElements)) {
  ...
}
```

Reply:
Alright, I'll make the changes and push asap.